client/src/App.js

```diff
@@ -5,22 +5,27 @@ import Feed from "./components/feed";
 import Signup from "./Component/SignUp/SignUp";
 import Signin from "./Component/SignIn/SignIn";
 import Homepage from "./Component/homepage/homepage";
+import NotificationPage from "./Component/NotificationPage/NotificationPage";
 import AddAQuestion from "./Component/AddAQuestion/AddAQuestion";
 
 const App = () => {
   return (
     <>
-      <BrowserRouter>
-        <Switch>
-          <Route path="/" exact component={Homepage} />
-          <Route path="/signup" component={Signup} />
-          <Route path="/signin" component={Signin} />
-          <Route path="/feed" component={Feed} />
-          <Route path="/addaquestion" component={AddAQuestion} />
-        </Switch>
-      </BrowserRouter>
-    </>
+
+  <BrowserRouter>
+      <Switch>
+        <Route path="/" exact component={Homepage} />
+        <Route path="/signup"  component={Signup} />
+        <Route path="/signin"  component={Signin} />
+        <Route path="/feed"  component={Feed} />
+        <Route path="/notification" component={NotificationPage} />
```

Comment:
Don't modify any other lines...or copy-paste the same thing...If you're adding a notification page line add that particular line only. I hope you're understanding what I'm trying to tell you. These small things help you in future😊

Reply:
Yea got it,
surely I will keep that in mind from next time👍
Thank you!